packages/camera/camera_android_camerax/android/src/main/java/io/flutter/plugins/camerax/CameraXProxy.java

```diff
@@ -23,4 +27,17 @@ public DeviceOrientationManager createDeviceOrientationManager(
       DeviceOrientationManager.DeviceOrientationChangeCallback callback) {
     return new DeviceOrientationManager(activity, isFrontFacing, sensorOrientation, callback);
   }
+
+  public Preview.Builder createPreviewBuilder() {
+    return new Preview.Builder();
+  }
+
+  public Surface createSurface(SurfaceTexture surfaceTexture) {
```

Comment:
Does this team have a stance on using @NonNull and @Nullable?

Reply:
We don't have an enforced repo wide policy yet. Mainly because we don't run a linter for Java code. I just created https://github.com/flutter/flutter/issues/119836 to track this.

For `webview_flutter_android`, `@Nonnull` is assumed and never used. But if something is nullable, then I use `@Nullable`.

I mostly recommend to be consistent whichever way you choose for the package. But, a consistent repo wide policy could be chosen when we start running a linter for Java code.

Reply:
For now lets lean towards documenting Nullable. 
Here is the internal guidance https://engdoc.corp.google.com/eng/doc/devguide/java/practices/null.md?cl=head&polyglot=java#annotations but I think in a public project that labeling both is probably better than relying on assumptions.

Reply:
Sounds good to me. I'll start adding it from here on out and most likely do a cleanup PR for this in the future to achieve consistency in the plugin.